src/panels/config/devices/ha-config-device-page.ts

```diff
@@ -161,26 +161,42 @@ export class HaConfigDevicePage extends LitElement {
     }
   );
 
+  private _getEntitiesSorted = (entities) =>
+    entities
+      .map((entity) => ({
+        ...entity,
+        stateName: this._computeEntityName(entity),
```

Comment:
`_computeEntityName` takes a `EntityRegistryEntry`, you are passing state object to it. Please add proper typing so these things pop up automatically.